build_tools/packaging/wheel/build_from_source.sh

```diff
@@ -28,6 +28,8 @@ popd
 
 # build mad, statically
 pushd libmad-0.15.1b
+# See https://stackoverflow.com/a/12864879/23845
```

Comment:
if you want, you can use: https://github.com/conda-forge/mad-feedstock/blob/master/recipe/mad.patch

I had issues with clang  as well, which need this

Reply:
I was able to build wheels after only this change, so that's kind of interesting.